examples/v201/central_system.py

```diff
@@ -65,8 +65,10 @@ async def on_connect(websocket, path):
 
     charge_point_id = path.strip('/')
     charge_point = ChargePoint(charge_point_id, websocket)
-
-    await charge_point.start()
+    try:
+        await charge_point.start()
+    except Exception as e:
+        logging.exception(e)
```

Comment:
logging.exception already contains the stack trace alongside.
```
In [1]: def start(): 
   ...:     raise KeyError 


In [6]: try: 
   ...:     start() 
   ...: except Exception: 
   ...:     logging.exception("Something happened") 
   ...:                                                                                             
ERROR:root:Something happened
Traceback (most recent call last):
  File "<ipython-input-6-edb1dc331440>", line 2, in <module>
    start()
  File "<ipython-input-1-190264bc2ae3>", line 2, in start
    raise KeyError
KeyError
```
So you can use that and add some custom message about it ;)